SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialLoadBalanceEnforcer.cpp

```diff
@@ -110,7 +110,7 @@ void USpatialLoadBalanceEnforcer::ProcessQueuedAclAssignmentRequests()
 			// TODO(zoning): Not sure whether this should be possible or not. Remove if we don't see the warning again.
 			UE_LOG(LogSpatialLoadBalanceEnforcer, Warning, TEXT("(%s) Entity without AuthIntent component will not be processed. EntityId: %lld"), *WorkerId, Request.EntityId);
 			CompletedRequests.Add(Request.EntityId);
-			return;
```

Comment:
Can you give any more details about when you're seeing this occur?

Reply:
Seems to happen when moving over boundaries...
```
LogSpatialActorChannel: Delete entity request on 1008. Has authority: 0
LogSpatialActorChannel: Delete entity request on 1008. Has authority: 0
LogSpatialActorChannel: Delete entity request on 9008. Has authority: 0
LogSpatialActorChannel: Delete entity request on 9008. Has authority: 0
LogSpatialActorChannel: Delete entity request on 1009. Has authority: 0
LogSpatialActorChannel: Delete entity request on 1009. Has authority: 0
LogSpatialLoadBalanceEnforcer: Warning: (UnrealWorker3BA423B24905C3B9E6176AB5ABE3B852) Entity without AuthIntent component will not be processed. EntityId: 1008
LogSpatialLoadBalanceEnforcer: Warning: (UnrealWorker3BA423B24905C3B9E6176AB5ABE3B852) Entity without AuthIntent component will not be processed. EntityId: 9008
LogSpatialLoadBalanceEnforcer: Warning: (UnrealWorker3BA423B24905C3B9E6176AB5ABE3B852) Entity without AuthIntent component will not be processed. EntityId: 1009
```

Reply:
This looks good.  Also looks like this warning is normal if the entity is being deleted.  Maybe we should additionally change it to a log